pulp_smash/pulp3/pytest_plugin/__init__.py

```diff
@@ -257,6 +262,57 @@ def cli_client(pulp_cfg):
     return cli.Client(pulp_cfg)
 
 
+@pytest.fixture(scope="session")
+def svc_mgr(pulp_cfg):
+    return cli.ServiceManager(pulp_cfg, PULP_HOST)
+
+
+@pytest.fixture
```

Comment:
I don't want to block this PR, but I'm not sure I understand the benefit of having them be pytest fixtures?  What does the fixture decoration provide us here

Reply:
+1